core/vm/interpreter.go

```diff
@@ -39,6 +39,13 @@ type Config struct {
 	// may be left uninitialised and will be set to the default
 	// table.
 	JumpTable [256]operation
+
+	// Enable support for EWASM contracts
+	EWASMEnabled bool
```

Comment:
Whether ewasm is enabled or not is not a configuration value, it's a chain fork rule.